doc/api/url.md

```diff
@@ -115,10 +116,12 @@ No decoding of the `path` is performed.
 
 The `query` property is either the "params" portion of the query string (
 everything *except* the leading ASCII question mark (`?`), or an object
-returned by the `querystring` module's `parse()` function:
+returned by the [`querystring`][] module's `parse()` function:
 
 For example: `'query=string'` or `{'query': 'string'}`
 
+No decoding of the query string is performed.
```

Comment:
This needs to clarified to say that this is only the case when the query string *isn't* parsed. When it is parsed, decoding *is* performed.